mkosi

```diff
@@ -2605,10 +2607,9 @@ def parse_args() -> CommandLineArguments:
     group.add_argument("--build-package", action=CommaDelimitedListAction, dest='build_packages', default=[], help='Additional packages needed for build script', metavar='PACKAGE')
     group.add_argument("--postinst-script", help='Postinstall script to run inside image', metavar='PATH')
     group.add_argument("--finalize-script", help='Postinstall script to run outside image', metavar='PATH')
-    group.add_argument('--use-git-files', type=parse_boolean,
-                       help='Ignore any files that git itself ignores (default: guess)')
+    group.add_argument("--copy-sources-method", choices=('plain', 'git'), help="Method used to copy build sources to the build image. 'plain': normal file copy; 'git': use git-ls-files, ignoring any file that git itself ignores. (default: guess)")
     group.add_argument('--git-files', choices=('cached', 'others'),
-                       help='Whether to include untracked files (default: others)')
+                       help='Whether to include untracked files, used together with --copy-sources-method=git (default: others)')
```

Comment:
if you clean this up with --copy-sources-method, maybe we can fold this in too, i.e. introduce --copy-sources-method=git-cached vs. --copy-sources-method=git-others?

I think fewer options are nice, in particular if they can be safely flattened into one enum.